debian/libibverbs1.symbols

```diff
@@ -4,7 +4,7 @@ libibverbs.so.1 libibverbs1 #MINVER#
  IBVERBS_1.1@IBVERBS_1.1 1.1.6
  IBVERBS_1.5@IBVERBS_1.5 20
  IBVERBS_1.6@IBVERBS_1.6 24
- (symver)IBVERBS_PRIVATE_22 22
+ (symver)IBVERBS_PRIVATE_25 25
```

Comment:
This is missing lines, be sure to test it with 'buildlib/cbuild pkg bionic':

```
dpkg-gensymbols: warning: some new symbols appeared in the symbols file: see diff output below
dpkg-gensymbols: warning: debian/libibverbs1/DEBIAN/symbols doesn't match completely debian/libibverbs1.symbols
--- debian/libibverbs1.symbols (libibverbs1_25.0-1_amd64)
+++ dpkg-gensymbolsEhYGYT	2019-07-12 20:00:15.906417218 +0000
@@ -4,6 +4,7 @@
  IBVERBS_1.1@IBVERBS_1.1 1.1.6
  IBVERBS_1.5@IBVERBS_1.5 20
  IBVERBS_1.6@IBVERBS_1.6 24
+ IBVERBS_1.7@IBVERBS_1.7 25.0-1
  (symver)IBVERBS_PRIVATE_25 25
  ibv_ack_async_event@IBVERBS_1.0 1.1.6
  ibv_ack_async_event@IBVERBS_1.1 1.1.6
@@ -89,6 +90,7 @@
  ibv_read_sysfs_file@IBVERBS_1.0 1.1.6
  ibv_reg_mr@IBVERBS_1.0 1.1.6
  ibv_reg_mr@IBVERBS_1.1 1.1.6
+ ibv_reg_mr_iova@IBVERBS_1.7 25.0-1
  ibv_register_driver@IBVERBS_1.1 1.1.6
  ibv_rereg_mr@IBVERBS_1.1 1.2.1
  ibv_resize_cq@IBVERBS_1.0 1.1.6
```

Reply:
So looks like 25 is already taken, just rebased and saw it.

To be honest, i'm totally lost here, any help is appreciated.

1. Do i still need to bump up BAPI?
2. Do i still need this section:
IBVERBS_1.7 {
global:
ibv_reg_mr_iova;
} IBVERBS_1.6;
3. Do i need to change this in libibverbs.map.in
1 1.7.${PACKAGE_VERSION}
4. Do i need to add this to debian/libibverbs1.symbols
ibv_reg_mr_iova@IBVERBS_1.7 25

Yuval